packages/cli/src/command_register.ts

```diff
@@ -48,7 +48,7 @@ const createOptionString = (
   isNegation = false
 ): string => {
   const actualName = isNegation ? `${OPTION_NEGATION_PREFIX}${name}` : name
-  const aliasAndName = alias ? `-${alias}, --${actualName}` : `--${actualName}`
+  const aliasAndName = alias ? `-${alias}, --${actualName}` : `,--${actualName}`
```

Comment:
1. can we please add a comment explaining why this is done? this looks like a bug in our code so will be good to know it is intentional
2. nit: 
```suggestion
  const aliasAndName = alias ? `-${alias}, --${actualName}` : `, --${actualName}`
```